src/globus_sdk/services/search/data.py

```diff
@@ -221,6 +222,56 @@ def add_sort(
         return self
 
 
+class SearchQueryV1(utils.PayloadWrapper):
+    """
+    A specialized dict which has helpers for creating and modifying a Search
+    Query document. Replaces the usage of ``SearchQuery``.
+
+    :param q: The query string. Required unless filters are used.
+    :param limit: A limit on the number of results returned in a single page
+    :param offset: An offset into the set of all results for the query
+    :param advanced: Whether to enable (``True``) or not to enable (``False``) advanced
+        parsing of query strings. The default of ``False`` is robust and guarantees that
+        the query will not error with "bad query string" errors
+    :param filters: a list of filters to apply to the query
+    :param facets: a list of facets to apply to the query
+    :param post_facet_filters: a list of filters to apply after facet
+        results are returned
+    :param boosts: a list of boosts to apply to the query
+    :param additional_fields: additional data to include in the query document
+    """
+
+    def __init__(
+        self,
+        *,
+        q: str | utils.MissingType = utils.MISSING,
+        limit: int | utils.MissingType = utils.MISSING,
+        offset: int | utils.MissingType = utils.MISSING,
+        advanced: bool | utils.MissingType = utils.MISSING,
+        filters: list[dict[str, t.Any]] | utils.MissingType = utils.MISSING,
+        facets: list[dict[str, t.Any]] | utils.MissingType = utils.MISSING,
+        post_facet_filters: list[dict[str, t.Any]] | utils.MissingType = utils.MISSING,
+        boosts: list[dict[str, t.Any]] | utils.MissingType = utils.MISSING,
+        sorts: list[dict[str, t.Any]] | utils.MissingType = utils.MISSING,
```

Comment:
Sorry for not catching this earlier, but it's `sort`, not `sorts`.

```suggestion
        sort: list[dict[str, t.Any]] | utils.MissingType = utils.MISSING,
```